src/services/crawl-tx/provider.registry.ts

```diff
@@ -0,0 +1,43 @@
+import seiTxRegistryType from './registry-type/sei-network.json' assert { type: 'json' };
+import auraTxRegistryType from './registry-type/aura-network.json' assert { type: 'json' };
+import { chainIdConfigOnServer } from '../../common';
+import config from '../../../config.json' assert { type: 'json' };
+
+export interface IProviderRegistry {
+  cosmos: any;
+  ibc: any;
+  txRegistryType: any;
+  seiprotocol: any;
+  aura: any;
+}
+
+export async function getProviderRegistry(): Promise<IProviderRegistry> {
+  let ibc;
+  let cosmos;
+  let seiprotocol;
+  let aura;
+  let txRegistryType;
+  switch (config.chainId) {
+    case chainIdConfigOnServer.Atlantic2:
+      ({ ibc, cosmos, seiprotocol } = await import(
```

Comment:
cái ngoặc ngoài là để làm gì nhỉ?

Reply:
Phải có cái ngoặc ngoài ấy anh không thì nó sẽ bị lỗi 
TS2695: Left side of comma operator is unused and has no side effects.

Reply:
hmm có return ở dưới mà nhỉ

Reply:
À đấy là chỉ vào biến thì nó báo lỗi vậy còn chỉ vào dấu bằng thì nó báo lỗi như này anh, chắc do lỗi này nên nó mới báo lỗi trên.
ESLint: Parsing error: Declaration or statement expected. This '=' follows a block of statements, so if you intended to write a destructuring assignment, you might need to wrap the the whole assignment in parentheses.
Trước em đọc cái này nên em mới thêm hai cái ngoặc ấy

Reply:
ok